nativelib/src/main/scala/scala/scalanative/runtime/package.scala

```diff
@@ -67,7 +67,7 @@ package object runtime {
     val mainThread = Thread.currentThread()
     if (mainThread == null) {
       ffi.printf(
-        c"Scala Native Fatal Error: failed to initialize main java.lang.Thread\n"
+        c"ScalaNative Fatal Error: failed to initialize main java.lang.Thread\n"
```

Comment:
This one here would need a standard helper function binding if you want to remain 100% DRY.

Reply:
Added a binding in [bde61cf](https://github.com/scala-native/scala-native/pull/4310/commits/bde61cf2c3cf432b22ef51b552577a7864949c36)